keyboards/cool836A/config.h

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2021 Ohashi
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x0000
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Ohashi
+#define PRODUCT      cool836A
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 6
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D1, B5, B4, F4, B1, B6 }
+#define MATRIX_COL_PINS { F5, D0, B2, C6, D7, E6 }
```

Comment:
However, what do I do with this part?

I've already tried on [your suggestion](https://github.com/qmk/qmk_firmware/pull/11467#discussion_r553530347) on my local environment, and then I got an error like
```
tmk_core/common/keycode.h:74:24: error: excess elements in array initializer [-Werror]
 #define KC_TRANSPARENT 0x01

```

Reply:
I've decided to work by myself.
Chances are that I should consult planck's codes.

I'll appreciate it if you wait for a little more time.